packages/components/src/Chips/InternalChipMultiSelect.tsx

```diff
@@ -32,7 +33,9 @@ export function InternalChipMultiSelect({
             <InternalChip
               {...chip.props}
               active={isChipActive}
-              suffix={checkmarkIcon(isChipActive)}
+              {...(isChipActive
```

Comment:
We changed the functionality of InternalChip a bit to not show the suffix if we don't provide it, so this just changes the prop passing a bit to reflect that.